src/App.vue

```diff
@@ -2,6 +2,7 @@
   <v-app>
     <SecurityBanner />
     <ATATSideBar v-if="loginStatus" />
+    <RightSideDrawer v-if="loginStatus" />
```

Comment:
- [ ] Let's make it more generic and call it `SideDrawer`, then put it a property called `pagePosition`  This would enable us to use what you've done for the left/right/top or bottom.  Ideally, it would be good if this could replace our current left hand side drawer someday.